ui/admin/tests/acceptance/aliases/list-test.js

```diff
@@ -43,15 +50,32 @@ module('Acceptance | aliases | list', function (hooks) {
       type: 'org',
       scope: { id: 'global', type: 'global' },
     });
+    instances.scopes.project = this.server.create('scope', {
+      type: 'project',
+      scope: { id: instances.scopes.org.id, type: 'org' },
+    });
+    instances.target = this.server.create('target', {
+      scope: instances.scopes.project,
+    });
+    instances.alias = this.server.create('alias', {
+      scope: instances.scopes.global,
+      destination_id: null,
+    });
+    instances.aliasWithTarget = this.server.create('alias', {
+      scope: instances.scopes.global,
+      destination_id: instances.target.id,
+    });
     urls.globalScope = `/scopes/global`;
     urls.aliases = `${urls.globalScope}/aliases`;
-
+    urls.alias = `${urls.aliases}/${instances.alias.id}`;
+    urls.aliasWithTarget = `${urls.aliases}/${instances.aliasWithTarget.id}`;
     intl = this.owner.lookup('service:intl');
 
     authenticateSession({});
   });
 
   test('users can navigate to aliases with proper authorization', async function (assert) {
+    this.server.schema.aliases.all().destroy();
```

Comment:
non-blocking: Why are we destroying all aliases?

Reply:
Ah good question, those test cases also test the empty app state, before there were no aliases but for search & created some in `beforeEach` hook